Header header
string class_id
float32 score
Pose pose
